App.js

```diff
@@ -1,19 +1,51 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Image, Fonts, Button } from 'react-native';
+import { Font } from 'expo';
 
 export default class App extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+    });
+    this.setState({ fontLoaded: true }) ;
+  }
   render() {
     return (
       <View style={styles.container}>
-        <View style={{flexDirection: 'row', paddingTop: 30}}>
-          <View>
-            <View style={styles.pongRoundedRectangle}>
-              <Text style={styles.pongMatesText}>Pong</Text>
+          <View style = {{ height: '10%', flexDirection: 'row', justifyContent: 'space-around', alignContent: 'flex-start'}}>
+            <View style = {styles.topButtons}>
+              <Image source = {require('./assets/profile.png')} style = {styles.image}/>
+            </View>
+            <View style = {styles.topButtons}>
+            {
+              this.state.fontLoaded ? (
+                <Text style = {styles.faqButton}>?</Text>
+              ) : null
```

Comment:
You have to change the `null` value to something else. If `this.state.fontLoaded` is ever `false` the question mark won't show up